### PR TITLE
Fixes #15968: Use systemctl for rudder services management in docs

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/15968

### CODE REVIEW

File: src/reference/modules/administration/pages/server.adoc [hunk 1]
```diff
@@ -220,30 +222,34 @@ include::{partialsdir}/glossary/web-server-application.adoc[]
...
+systemctl restart httpd
 
 ----
 
 include::{partialsdir}/glossary/ldap-server.adoc[]
 
 ----
 
-service rudder-slapd restart
+systemctl restartrudder-slapd
```
Comment (peckpeck) [inline src/reference/modules/administration/pages/server.adoc]: missing space

File: src/reference/modules/administration/pages/server.adoc [hunk 2]
```diff
@@ -220,30 +222,34 @@ include::{partialsdir}/glossary/web-server-application.adoc[]
...
 
 ----
 
 include::{partialsdir}/glossary/sql-server.adoc[]
 
 ----
 
-service postgresql* restart
+# The name of the service may vary if you installed
+# non-standard postgresql packages
```
Comment (peckpeck) [inline src/reference/modules/administration/pages/server.adoc]: is there a command to list postgresql* ?

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**